src/routes/EResourceViewRoute.js

```diff
@@ -135,6 +132,17 @@ class EResourceViewRoute extends React.Component {
   handleFilterPackageContents = (path) => {
     const { mutator } = this.props;
     mutator.packageContentsFilter.replace(path);
+    mutator.resultOffset.replace(0);
+    mutator.resultCount.replace(resultCount.INITIAL_RESULT_COUNT);
+  }
+
+  handleNeedMorePackageContents = (_askAmount, index) => {
+    const { mutator, resources } = this.props;
+    if (index > 0) {
+      mutator.resultOffset.replace(index);
+    } else {
+      mutator.resultCount.replace(resources.resultCount + resultCount.RESULT_COUNT_INCREMENT);
```

Comment:
Does this case ever run? Aren't we always using the offset-fetch, so this onNeedMore callback will always have an `index`?

Reply:
Hmm, weird I dont think we even need `resultCount` here